File: SingularSystem.Api.Products.Domain/DTOs/ProductSaleSummary.cs
namespace SingularSystem.Api.Products.Domain.DTOs
{
    public class ProductSaleSummary
	{
		public int SaleId { get; set; }
		public int SaleQty { get; set; }
		public decimal SalePrice { get; set; }
		public DateTime SaleDate { get; set; }
	}
}

File: singularsystems.products.web/src/entities/products/ProductSaleSummary.js
export class ProductSaleSummary{
    constructor(jsonData){
        this.saleId = '';
        this.productId='';
        this.salePrice='';
        this.saleQty='';
        this.saleDate='';

        Object.assign(this,jsonData);
       
    }
}